server/controllers/finance/patientInvoice.js

```diff
@@ -234,8 +234,7 @@ function find(options) {
       em.text AS patientReference, service.name as serviceName, proj.name AS project_name,
       user.display_name, invoice.user_id, invoice.reversed, invoice.edited
     FROM invoice
-    LEFT JOIN patient ON invoice.debtor_uuid = patient.debtor_uuid
```

Comment:
I've never seen FORCE INDEX before.  From what I am reading, this looks like the right solution!

Reply:
@jniles it's now very fast... 460ms instead of 11seconds